hack/multi-node/bootkube-render

```diff
@@ -13,7 +13,4 @@ echo "inserting kubeconfig into user-data"
 cat user-data.sample > cluster/user-data && sed 's/^/      /' cluster/auth/kubeconfig.yaml >> cluster/user-data
 cp cluster/user-data{,-worker}
 cp cluster/user-data{,-controller}
-sed -i.bak \
-    -e 's/127\.0\.0\.1/172.17.4.101/' \
-    -e '/^.*master.*/d' \
-    cluster/user-data-worker
+sed -i '' -e '/--node-labels=master=true/d' cluster/user-data-worker
```

Comment:
Yeah, I think you're right. I'm with with keeping it to `-i.bak` or something and them `rm`ing that file afterwards.